tools/perf/brave_page_sets/data/system_health_mobile.json

```diff
@@ -23,6 +23,9 @@
         },
         "load:site:youtube:2023": {
             "DEFAULT": "system_health_mobile_31be13a52f.wprgo"
+        },
+        "multitab_load:site:example.com:2024": {
+            "DEFAULT": "system_health_desktop_8c7c864e79.wprgo"
```

Comment:
it's a wrong file.
should be `"system_health_mobile_31be13a52f.wprgo"`

Reply:
Changed it!